agent/fw_wordpress.c

```diff
@@ -578,6 +578,22 @@ NR_PHP_WRAPPER(nr_wordpress_apply_filters) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_wordpress_version() {
+  char* string = "$GLOBALS['wp_version'];";
+  zval retval;
+  int result = zend_eval_string(string, &retval,
+                                "Retrieve Wordpress Version");
+  
+  // Add php package to transaction
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      nr_txn_add_php_package(NRPRG(txn), "wordpress", version);
+      zval_dtor(&retval);
+    }
```

Comment:
```suggestion
    }
    zval_dtor(&retval);
```
Even if the version is somehow not a string, we want to free the zval